pkg/platform/packet/template.go

```diff
@@ -23,6 +23,8 @@ module "packet-{{.Config.ClusterName}}" {
   ssh_keys  = {{.SSHPublicKeys}}
   asset_dir = "../cluster-assets"
 
+  auth_token = "{{.Config.AuthToken}}"
```

Comment:
This variable is not exposed. Perhaps this commit should be merged with the other one which adds the variable.